Fix svnauthz_tests.py by adding /usr/bin to PATH.

From: David Rothenberger <daveroth@acm.org>


---
 subversion/tests/cmdline/svnauthz_tests.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/subversion/tests/cmdline/svnauthz_tests.py b/subversion/tests/cmdline/svnauthz_tests.py
index fd1de40..4ced90d 100755
--- a/subversion/tests/cmdline/svnauthz_tests.py
+++ b/subversion/tests/cmdline/svnauthz_tests.py
@@ -48,7 +48,7 @@ svnauthz_bin=%s
 fp = open(os.path.join(sys.argv[1], 'hooks.log'), 'wb')
 def output_command(fp, cmd, opt):
   command = [svnauthz_bin, cmd, '-t', sys.argv[2], sys.argv[1]] + opt
-  process = subprocess.Popen(command, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=False, bufsize=-1)
+  process = subprocess.Popen(command, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=False, bufsize=-1, env={'PATH': '/usr/bin'})
   (output, errors) = process.communicate()
   status = process.returncode
   fp.write(output)
